src/Form/LegacyConsumer/resources/js/conditinal-fields.js

```diff
@@ -5,6 +5,27 @@ document.addEventListener('readystatechange', event => {
 
 	const state = {};
 
+	/**
+	 * @unreleased
+	 *
+	 * @param inputField
+	 * @return {*}
```

Comment:
I think this should always produce a string?

```suggestion
	 * @param {HTMLElement} inputField
	 * @return {string}
```